### ISSUE TITLE
Addon not functioning properly

### ISSUE THREAD

**Issue by isabellaalstrom** [hassio-addons/addon-grocy#300]
# Problem/Motivation

The addon does not work

## Expected behavior

It should work 😀

## Actual behavior
Addon logs says:

```
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
fix-attrs: info: applying /etc/fix-attrs.d/01-grocy
s6-chown: fatal: unable to chown /data/grocy/storage/recipepictures/jw3xiubu3k8z3dt4bntyulKoreansk: No such file or directory
fix-attrs: warning: fix-attrs is deprecated, please fix volume permissions in your container manager instead
fix-attrs: warning: some fix files failed to apply
s6-rc: warning: unable to start service fix-attrs: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

## Steps to reproduce

I updated from 0.16.0 to 0.18.0.

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
Crab! 🦀

**Comment by isabellaalstrom** [comment]
I tried 0.17.0 also before, and something didn't work then either, but I thought it had something to do with the component and I didn't have the time then, so I just downgraded from my backup. But that means it might not be a .18-thing.

**Comment by frenck** [comment]
No this is definitely caused by the 0.18 release, as it is related to base image changes.

Looking for a solution 🤔

**Comment by SupremeSports** [comment]
Just to chime in, version 0.17 works but not 0.18, gives 502: Bad gateway

**Comment by SupremeSports** [comment]
@frenck nothing different, same error log

`s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
fix-attrs: info: applying /etc/fix-attrs.d/01-grocy
s6-chown: fatal: unable to chown /data/grocy/storage/productpictures/nnz602adhfu5hqy43xzos-l400: No such file or directory
fix-attrs: warning: fix-attrs is deprecated, please fix volume permissions in your container manager instead
fix-attrs: warning: some fix files failed to apply
s6-rc: warning: unable to start service fix-attrs: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped`

**Comment by dbaron92** [comment]
I have a same problem. This morning, I found out about Grocy, I decided to give it try (first time installation) and got this error log.
`s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
fix-attrs: info: applying /etc/fix-attrs.d/01-grocy
s6-chown: fatal: unable to chown /data/grocy: No such file or directory
fix-attrs: warning: fix-attrs is deprecated, please fix volume permissions in your container manager instead
fix-attrs: warning: some fix files failed to apply
s6-rc: warning: unable to start service fix-attrs: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped`

**Comment by KyleStilkey** [comment]
s6-rc: info: service s6rc-oneshot-runner: starting
s6-rc: info: service s6rc-oneshot-runner successfully started
s6-rc: info: service fix-attrs: starting
fix-attrs: info: applying /etc/fix-attrs.d/01-grocy
s6-chown: fatal: unable to chown /data/grocy/storage/productpictures/790kbybau2ltp2741um3fgvegetable: No such file or directory
fix-attrs: warning: fix-attrs is deprecated, please fix volume permissions in your container manager instead
fix-attrs: warning: some fix files failed to apply
s6-rc: warning: unable to start service fix-attrs: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped

**Comment by frenck** [comment]
Just shipped v0.18.1 to address these issues. Hopefully I got them all with this change 🙏

**Comment by SupremeSports** [comment]
> Just shipped v0.18.1 to address these issues. Hopefully I got them all with this change 🙏

Seems to be working flawlessly :)

**Comment by frenck** [comment]
Cool! Thanks for letting me know @SupremeSports 👍
